apitools/base/py/base_api.py

```diff
@@ -614,10 +614,11 @@ def __ProcessHttpResponse(self, method_config, http_response, request):
             content = content.decode(self._client.response_encoding)
 
         if self.__client.response_type_model == 'json':
-            return content
+            return http_response.content
```

Comment:
I *think* this should go back to being `content` (since it could have had its type changed a couple lines up when it was decoded).  Same for line 621 below.

Reply:
Whoops, I'll fix this! So preferring Ed's version, right?